Client/Frontend/Browser/Toolbars/UrlBar/CollapsedURLBarView.swift

```diff
@@ -68,7 +68,7 @@ class CollapsedURLBarView: UIView {
   var currentURL: URL? {
     didSet {
       urlLabel.text = currentURL.map {
-        URLFormatter.formatURL(forDisplayOmitSchemePathAndTrivialSubdomains: $0.absoluteString)
+        URLFormatter.formatURLOrigin(forDisplayOmitSchemePathAndTrivialSubdomains: $0.absoluteString)
```

Comment:
Wondering If we should add `formatURLOrigin` to 
https://github.com/brave/brave-ios/blob/5611ad2babaaa848ad0bd873ca2c00c255cfb27f/Client/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift#L411

and maybe edit locationText

https://github.com/brave/brave-ios/blob/5611ad2babaaa848ad0bd873ca2c00c255cfb27f/Client/Frontend/Browser/Toolbars/UrlBar/TopToolbarView.swift#LL666C9-L666C9

Reply:
Lets do a separate PR for that

Reply:
Plus the API for that is going to be the new one (`formatURL`) not `formatURLOrigin`

Reply:
Sounds good, I will check in detail after we merge core update